### PR TITLE
Update cassandra.md

### PR DESCRIPTION
**Pull request by nmohantyctr**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/databases/cassandra.md [hunk 1]
```diff
@@ -525,6 +525,10 @@ At this point, Cassandra metrics and logs should start flowing into Sumo Logic.
 
 ## Installing Cassandra Monitors
 
+:::
+Note: This step is not needed if one is using application components solution terraform script.
+:::
```
Comment (kimsauce) [inline docs/integrations/databases/cassandra.md:530]: fix note formatting

File: docs/integrations/databases/cassandra.md [hunk 2]
```diff
@@ -609,6 +613,10 @@ email_notifications = [
 
 ## Installing the Cassandra App
 
+:::
+Note: This step is not needed if one is using application components solution terraform script.
```
Comment (kimsauce) [inline docs/integrations/databases/cassandra.md:617]: fix note formatting

**Review by kimsauce — CHANGES_REQUESTED** [review]
formatting fixes

File: docs/integrations/databases/cassandra.md (hunk 1 above)
Comment (nmohantyctr) [reply docs/integrations/databases/cassandra.md]: Fixed

File: docs/integrations/databases/cassandra.md (hunk 2 above)
Comment (nmohantyctr) [reply docs/integrations/databases/cassandra.md]: Fixed

File: docs/integrations/databases/cassandra.md [hunk 3]
```diff
@@ -525,6 +525,10 @@ At this point, Cassandra metrics and logs should start flowing into Sumo Logic.
 
 ## Installing Cassandra Monitors
 
+:::note
+This step is not needed if one is using application components solution terraform script.
```
Comment (kimsauce) [inline docs/integrations/databases/cassandra.md:529]:
```suggestion
This step is not needed if you are using the [Application Components Solution](/docs/observability/application-components) Terraform script.
```

File: docs/integrations/databases/cassandra.md [hunk 4]
```diff
@@ -609,6 +613,10 @@ email_notifications = [
 
 ## Installing the Cassandra App
 
+:::note
+This step is not needed if one is using application components solution terraform script.
```
Comment (kimsauce) [inline docs/integrations/databases/cassandra.md:617]:
```suggestion
This step is not needed if you are using the [Application Components Solution](/docs/observability/application-components) Terraform script.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**